### PR TITLE
[Preloader] Implement preloader

### PR DESCRIPTION
**Pull request by di-shu**
Preloader component takes one boolean argument to check is data loaded or no.
If true - preloader will disable.

### CODE REVIEW

**Review by Dmytro1991ua — COMMENTED** [review]
LGTM

File: src/hooks/usePreloader.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      preloader.remove();
+    }
+  }
+
+  useEffect(() => {
+    checkData();
+  })
+}
+
+export default usePreloader;
```
Comment (solomiia-sahala) [inline src/hooks/usePreloader.tsx:17]: pls, add extra line in the end

**Review by solomiia-sahala — APPROVED** [review]

**Review by Dmytro1991ua — APPROVED** [review]

**Review by Kachyr — APPROVED** [review]

File: src/hooks/usePreloader.tsx (hunk 1 above)
Comment (di-shu) [reply src/hooks/usePreloader.tsx:17]: There is, I don't know why it is hidden here.

File: src/hooks/usePreloader.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  const checkData = (): void => {
+    const preloader = document.querySelector('.preloader');
+
+    if(isLoaded && preloader) {
+      preloader.remove();
+    }
+  }
+
+  useEffect(() => {
+    checkData();
```
Comment (oleksandrstarov) [inline src/hooks/usePreloader.tsx:13]: maybe add a deps array with isLoaded? 
Also we need to check data only if isLoaded = true? 
The we can add check to useEffect, and rename function to removeLoader and not use isLoaded inside that function

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

**Review by oleksandrstarov — APPROVED** [review]
LGTM

**Pull request merged**